packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBarRow.tsx

```diff
@@ -437,6 +443,11 @@ export class UnthemedSpanBarRow extends React.PureComponent<SpanBarRowProps> {
               </span>
               <small className={styles.endpointName}>{rpc ? rpc.operationName : operationName}</small>
               <small className={styles.endpointName}> | {label}</small>
+              {references.find((ref) => ref.refType === 'FOLLOWS_FROM') && (
```

Comment:
This is probably only part I would leave out for now. Reason is we have other icons here (to logs) which is actually a link. Having non clickable icon next to it is probably confusing. At the same time as there can be multiple links we would need a dropdown menu in case of multiple refs (which would probably make sense also for the log links). In any case probably would make sense as a separate PR.

Reply:
Agreed we should revisit the UI a bit to consider how to handle 'n' links. Trace to logs was 0 to 1, now going from 1 to 2 it makes sense to consider how to handle any number as we'll likely have some more internal linking use cases soon.